bedrock/firefox/templates/firefox/campaign.html

```diff
@@ -0,0 +1,74 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% add_lang_files "firefox/campaign" %}
```

Comment:
@peiying2 hi, I have a slightly unusual request here.

This PR is adding a new `/firefox/campaign/` page, but the strings are all from the current `/firefox/new/` page. The idea is that this new page will be completely independent to `/new`, so we would like to create a duplicate l10n file for it. There is no requirement to translate any new strings, but we would like to copy all the strings from `firefox/new/quantum` to this new lang file.

Is this something you might be able to help us with? I could do it manually and submit a PR to the l10n repo, but there might be a quicker way I don't know about?

Reply:
I opened a pull request for the langfile here https://github.com/mozilla-l10n/www.mozilla.org/pull/277